### DISCUSSION TITLE
Chinese literature

### DISCUSSION THREAD

**Discussion by LEEDENGS** [arc53/DocsGPT#766] [Q&A]
Hello dear developers, I am a law undergraduate from China, and when using your amazing product, I find that there seems to be limited support for Chinese literature. I don't know if it's my problem? Thank you again for developing such a great program

**Comment by dartpain** [comment]
Can you send your source doc and some exaples of the questions you are asking please

**Reply by LEEDENGS** [reply]
Hello! Thank you so much for answering my problem.
There are two main aspects to this problem.
First of all, after I uploaded a Chinese document, I asked this AI assistant, and she answered like this

_很抱歉，我无法读取您提供的文章内容。作为一个AI助手，我无法阅读并理解外部文章。如果您能提供有关文章的简要摘要或主题，我将尽力为您提供帮助和答案。
I'm sorry, but I can't read the content of the article you provided. As an AI assistant, I can't read and understand external articles. If you can 
provide a brief summary or topic about the article, I will do my best to help and answer for you._

I tried the next English article, and she could answer the questions based on the content of the literature, and I thought the answers were quite efficient.

**Reply by LEEDENGS** [reply]
[美国离婚后的子女监护制度及其启示_王丽萍 (2).pdf](https://github.com/arc53/DocsGPT/files/13331621/_.2.pdf)
Thank you and your team for developing the product, this is a Chinese legal literature which I prefer in my paper, forgive me for not knowing code and AI to describe my problem further， thank you again for your help!
